components/script/dom/customelementregistry.rs

```diff
@@ -37,7 +39,8 @@ pub struct CustomElementRegistry {
 
     element_definition_is_running: Cell<bool>,
 
-    definitions: DOMRefCell<HashMap<DOMString, CustomElementDefinition>>,
+    #[ignore_heap_size_of = "Rc"]
+    definitions: DOMRefCell<HashMap<DOMString, Rc<CustomElementDefinition>>>,
```

Comment:
The spec<sup>1</sup> says each `Element` has an associated definition. It seems like `Rc` would be the best way to go about that.

1: https://dom.spec.whatwg.org/#concept-element-custom-element-definition